bigtable/filters/filters_test.go

```diff
@@ -0,0 +1,602 @@
+				`Reading data for phone#4c410523#20190501:
+Column Family cell_plan
+	data_plan_01gb: false @%[1]d
+	data_plan_01gb: true @%[2]d
+	data_plan_05gb: true @%[1]d
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 1 @%[1]d
+	os_build: PQ2A.190405.003 @%[1]d
+
+Reading data for phone#4c410523#20190502:
+Column Family cell_plan
+	data_plan_05gb: true @%[1]d
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 1 @%[1]d
+	os_build: PQ2A.190405.004 @%[1]d
+
+Reading data for phone#4c410523#20190505:
+Column Family cell_plan
+	data_plan_05gb: true @%[1]d
+Column Family stats_summary
+	connected_cell: 0 @%[1]d
+	connected_wifi: 1 @%[1]d
+	os_build: PQ2A.190406.000 @%[1]d
+
+Reading data for phone#5c10102#20190501:
+Column Family cell_plan
+	data_plan_10gb: true @%[1]d
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 1 @%[1]d
+	os_build: PQ2A.190401.002 @%[1]d
+
+Reading data for phone#5c10102#20190502:
+Column Family cell_plan
+	data_plan_10gb: true @%[1]d
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 0 @%[1]d
+	os_build: PQ2A.190406.000 @%[1]d`, timestamp, timestampMinusHr),},
+		{
+			name:   "filterLimitCellsPerRow",
+			filter: filterLimitCellsPerRow,
+			want: fmt.Sprintf(
+				`Reading data for phone#4c410523#20190501:
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 1 @%[1]d
+
+Reading data for phone#4c410523#20190502:
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 1 @%[1]d
+
+Reading data for phone#4c410523#20190505:
+Column Family stats_summary
+	connected_cell: 0 @%[1]d
+	connected_wifi: 1 @%[1]d
+
+Reading data for phone#5c10102#20190501:
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 1 @%[1]d
+
+Reading data for phone#5c10102#20190502:
+Column Family stats_summary
+	connected_cell: 1 @%[1]d
+	connected_wifi: 0 @%[1]d`, timestamp, timestampMinusHr),},
+		{
+			name:   "filterLimitCellsPerRowOffset",
+			filter: filterLimitCellsPerRowOffset,
+			want: fmt.Sprintf(
+				`Reading data for phone#4c410523#20190501:
+Column Family cell_plan
+	data_plan_01gb: false @%[1]d
+	data_plan_01gb: true @%[2]d
+	data_plan_05gb: true @%[1]d
+Column Family stats_summary
+	os_build: PQ2A.190405.003 @%[1]d
+
+Reading data for phone#4c410523#20190502:
+Column Family cell_plan
+	data_plan_05gb: true @%[1]d
+Column Family stats_summary
+	os_build: PQ2A.190405.004 @%[1]d
+
+Reading data for phone#4c410523#20190505:
+Column Family cell_plan
+	data_plan_05gb: true @%[1]d
+Column Family stats_summary
+	os_build: PQ2A.190406.000 @%[1]d
+
+Reading data for phone#5c10102#20190501:
+Column Family cell_plan
+	data_plan_10gb: true @%[1]d
+Column Family stats_summary
+	os_build: PQ2A.190401.002 @%[1]d
+
+Reading data for phone#5c10102#20190502:
+Column Family cell_plan
+	data_plan_10gb: true @%[1]d
+Column Family stats_summary
+	os_build: PQ2A.190406.000 @%[1]d`, timestamp, timestampMinusHr),},
```

Comment:
Please put the `},` on its own line. Makes it slightly easier to spot the new test case.